common/web/types/src/consts/modifier-key-constants.ts

```diff
@@ -6,9 +6,14 @@
 
 import { VisualKeyboardShiftState } from "../kvk/visual-keyboard.js";
 
+/**
+ * This type is declared as a `const` rather than as an `enum` for
+ * historical reasons. Use instead `ModifierKeyConstant` where possible.
```

Comment:
I guess you mean we should use `ModifierKeyConstant` instead of `ModifierKeyConstants`. I first misread it the other direction, so it might be good to make it a bit clearer, e.g.

```suggestion
 * historical reasons. Use `ModifierKeyConstant` instead where possible.
```